internal/build/extend_build_test.go

```diff
@@ -0,0 +1,160 @@
+package build_test
+
+import (
+	"bytes"
+	"context"
+	"io"
+
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/apex/log"
+
+	"github.com/buildpacks/pack/internal/build"
+	"github.com/buildpacks/pack/internal/build/fakes"
+
+	"github.com/docker/docker/api/types"
+	"github.com/golang/mock/gomock"
+	"github.com/heroku/color"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/pack/pkg/archive"
+	"github.com/buildpacks/pack/pkg/logging"
+	h "github.com/buildpacks/pack/testhelpers"
+
+	mockdocker "github.com/buildpacks/pack/internal/build/mockdocker"
```

Comment:
```suggestion
	"bytes"
	"context"
	"io"
	"path/filepath"
	"strconv"
	"strings"
	"testing"

	"github.com/apex/log"
	"github.com/docker/docker/api/types"
	"github.com/golang/mock/gomock"
	"github.com/heroku/color"
	"github.com/sclevine/spec"
	"github.com/sclevine/spec/report"

	"github.com/buildpacks/pack/pkg/archive"
	"github.com/buildpacks/pack/pkg/logging"
	h "github.com/buildpacks/pack/testhelpers"
	"github.com/buildpacks/pack/internal/build"
	"github.com/buildpacks/pack/internal/build/fakes"
	mockdocker "github.com/buildpacks/pack/internal/build/mockdocker"
```
This groups the imports by go standard library, external, and internal - you might need to additionally sort them

Reply:
done!!